domain-services/fiscal-event-service/src/main/resources/db/migration/main/V20220728121459__DDL_fiscal_event_create_ddl.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE eg_ifix_fiscal_event (
+    id varchar,
+    tenantid varchar NOT NULL,
+    sender varchar NOT NULL,
+    receivers JSONB,
```

Comment:
Why receivers is a JSONB? It is a list of strings in the API contract. 
It can be a table linking a fiscal event with a list of receivers. 
We need search queries on receivers so keeping it as JSONB would also have efficiency issues.